drivers/SmartThings/zigbee-presence-sensor/src/arrival-sensor-v1/init.lua

```diff
@@ -111,19 +111,18 @@ end
 
 local function init_handler(self, device, event, args)
   device:set_field(
-    timer_const.PRESENCE_CALLBACK_CREATE_FN,
+    presence_utils.PRESENCE_CALLBACK_CREATE_FN,
     function(device)
       return device.thread:call_with_delay(
               3 * CHECKIN_INTERVAL + 1,
               function()
                 device:emit_event(PresenceSensor.presence("not present"))
-                device:emit_event(SignalStrength.lqi(0))
-                device:emit_event(SignalStrength.rssi({value = -100, unit = 'dBm'}))
```

Comment:
these signal strength events should have been previously removed when the capabilities were removed from the profile, but this case was missed.